test/Hpack/ConfigSpec.hs

```diff
@@ -642,6 +645,13 @@ spec = do
         }
         )
 
+    it "allows overriding fields and takes the last value" $ do
+      withPackageConfig_ [i|
+        name: n1
+        name: n2
+        |]
+        (packageName >>> (`shouldBe` "n2"))
```

Comment:
My assumption was that this will always work and is not violating the spec.  But then again, I'm not sure.